rpcserver.go

```diff
@@ -2257,13 +2257,6 @@ func (r *rpcServer) CommitVirtualPsbts(ctx context.Context,
 		return nil, fmt.Errorf("error funding packet: %w", err)
 	}
 
-	// Validate the actual fee rate of the transaction.
-	txFees, err := fundedPacket.GetTxFee()
-	if err != nil {
-		return nil, fmt.Errorf("error calculating transaction fees: %w",
-			err)
-	}
-
```

Comment:
Yes this was intentional, the `txFees` was only used to set a field in the `tapsend.AnchorTransaction` we populated below. But since we're only going to use the `fundedPacket.UnsignedTx` directly, everything else is not needed anymore.